### PR TITLE
Docs reference correct cli commands for versioned rzup install for v1.1+

### PR DESCRIPTION
**Pull request by nuke-web3**
docs lagged the present state-of-the-art for `rzup` to change the versions. This amends the few places it was wrong for the latest version of the tool.

Related to https://github.com/risc0/risc0/issues/2382 and https://github.com/risc0/risc0/pull/2372

### CODE REVIEW

**Review by SchmErik — APPROVED** [review]
looks good to me. Thanks for the fix!

**Review by hmrtn — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -60,19 +62,19 @@ cargo xtask gen-receipt
...
-1. Tests pass with:
-    ```
-    cargo test -F prove -F docker
-    $(cd examples && cargo test)
-    $(cd benchmarks && cargo test)
-    ```
-1. Commit any outdated `Cargo.lock` files.
-1. Open a PR against the `main` branch.
+2. Format with `cargo fmt --all`.
+3. Lint with:
```
Comment (flaub) [inline CONTRIBUTING.md:66]: These should auto-number when it renders, so leaving `1.` should work

Comment (nuke-web3) [reply CONTRIBUTING.md:66]: This was from running the preferred formatter for docs on all the md I touched: 

https://github.com/risc0/risc0/blob/f3a78d6f4babac2106e64e3d731f348b75735fee/website/package.json#L16-L18
https://github.com/risc0/risc0/blob/f3a78d6f4babac2106e64e3d731f348b75735fee/website/.remarkrc#L39-L50

I know that manual numbering is the strong preference for @nategraf that might wanna chime in here in gerenal (we can make a new issue if we feel being uniform in how to use md ordered lists is worthwhile)

image

File: website/api/zkvm/install.md [hunk 2]
```diff
@@ -53,3 +53,4 @@ After you update your installation, be sure to update your project's RISC Zero c
 [release tag]: https://github.com/risc0/risc0/releases
 [Rust]: https://www.rust-lang.org
 [rustup]: https://rustup.rs
+[rzup-repo]: https://github.com/risc0/risc0/tree/release-1.1/rzup
```
Comment (flaub) [inline website/api/zkvm/install.md:56]:
```suggestion
[rzup-repo]: https://github.com/risc0/risc0/tree/main/rzup
```

Since this is for the 'next' version

**Review by flaub — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (flaub) [reply CONTRIBUTING.md:66]: OK, not a huge deal either way

**Review by pdg744 — APPROVED** [review]

**Pull request merged**